data/netcdf_data.py

```diff
@@ -53,6 +54,7 @@ def __enter__(self):
         if not self.meta_only:
             # Don't decode times since we do it anyways.
             decode_times = False
+            url = self.url if not isinstance(self.url, list) else self.url[0]
```

Comment:
Is this zarr specific? Probably should be incorporated into line 71 directly.  Careful -- what would happen if "url" was a list?

Reply:
Yes, It's specific to Zarr.
Now it's added into line 71 (New line:69).
If the URL is given as a list for other file formats, IF condition will handle it, hope Zarr file path won't be a list